lib/mirage/mirage.mli

```diff
@@ -267,14 +267,15 @@ val kv_ro : kv_ro typ
 (** Implementations of the [Mirage_kv.RO] signature. *)
 
 val crunch : string -> kv_ro impl
-(** Crunch a directory. *)
+(** Crunch a directory. The input directory is compiled into a memory-backed
+    copy. *)
```

Comment:
I'd elaborate on that:
Crunch a directory. The contents of the directory is transformed into OCaml code, which is then compiled as part of the unikernel.